sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -0,0 +1,938 @@
+| `matrix_type::uint8`  | `matrix_type::int8` |
+`matrix_type::sint32`  |  +<=+ 16 |  +<=+ 16 |  +<=+ 64
+| `matrix_type::int8`  | `matrix_type::uint8` |
+`matrix_type::sint32`  |  +<=+ 16 |  +<=+ 16 |  +<=+ 64
+| `matrix_type::int8`  | `matrix_type::int8` |
+`matrix_type::sint32`  |  +<=+ 16 |  +<=+ 16 |  +<=+ 64
+|  `matrix_type::bf16`       |  `matrix_type::bf16`   |
+`matrix_type::fp32`   |  +<=+ 16 |  +<=+ 16   |  +<=+ 32
+|======================
+
+==== Intel XMX Supported Combinations
+This is currently available in devices with the architecture
+`architecture::intel_gpu_pvc` and `architecture::intel_gpu_dg2`. In
+these architectures' implementation, the type of the C matrix must be
+the same as the type of the D matrix. Therefore, that common type is
+shown in a single column in the table below.
+
+[frame="none",options="header"]
+|======================
+| A type | B type | C and D type | M | N | K | device
+| `matrix_type::uint8`  | `matrix_type::uint8` |
+`matrix_type::int32`  |  +<=+ 8 |  16 |  32 | architecture::intel_gpu_pvc
+| | | | |8||architecture::intel_gpu_dg2
+| `matrix_type::uint8`  | `matrix_type::int8` |
+`matrix_type::int32`  |  +<=+ 8 |  16 |  32 | architecture::intel_gpu_pvc
+| | | | |8||architecture::intel_gpu_dg2
+| `matrix_type::int8`  | `matrix_type::uint8` |
+`matrix_type::int32`  |  +<=+ 8 |  16 |  32 | architecture::intel_gpu_pvc
+| | | | |8||architecture::intel_gpu_dg2
+| `matrix_type::int8`  | `matrix_type::int8` |
+`matrix_type::int32`  |  +<=+ 8 |  16 |  32 | architecture::intel_gpu_pvc
+| | | | |8||architecture::intel_gpu_dg2
+|  `matrix_type::fp16`       |  `matrix_type::fp16`   |
+`matrix_type::fp32`   |  +<=+ 8 |  16   |  16 | architecture::intel_gpu_pvc
+| | | | |8|| architecture::intel_gpu_dg2
+|  `matrix_type::bf16`       |  `matrix_type::bf16`   |
+`matrix_type::fp32`   |  +<=+ 8 |  16   |  16 | architecture::intel_gpu_pvc
+| | | | |8|| architecture::intel_gpu_dg2
+|======================
+
+==== Nvidia Tensor Cores Supported Combinations
+The complete set of matrix data types and shapes that are supported by
+the `ext_oneapi_cuda` backend are represented in the following
+table. In this architecture's implementation,
+the type of the A matrix must be the same as the type of the B
+matrix. Also, the type of the C matrix must be the same as the type of the D
+matrix.
+
+IMPORTANT: When compiling for the `ext_oneapi_cuda` backend the target
+arch backend flag, `-Xsycl-target-backend --cuda-gpu-arch=sm_xx`, must
+be used, where `sm_xx` must be a Compute Capability that is equal to
+or greater than the appropriate Minimum Compute Capability. When an
+executable has been compiled for `sm_xx`, if the executable is run on
+a device with compute capability less than `sm_xx` then an error will
+be thrown. The mapping to Minimum Compute Capability from each
+supported parameter combination is specified in the following table.
+
+
+[frame="none",options="header"]
+[cols="40%, 60%"]
```

Comment:
This line was added in a7f92ce2dc006914878f8dcc913afcb9a8a97213.  This change does not make sense because you are defining column widths for two columns, but this is a 6-column table.  In addition, the column widths were fine for this table, so there is no need to adjust them.  Please remove this line.